### PR TITLE
fix: Fix retry delay

### PR DESCRIPTION
**Pull request by dbrasseur-aneo**

### CODE REVIEW

File: Client/src/Common/Properties.cs [hunk 1]
```diff
@@ -285,4 +324,19 @@ public string ConnectionString
...
+
+  /// <summary>
+  ///   Backoff multiplier for retries
+  /// </summary>
+  public double RetryBackoffMultiplier { get; set; } = 2;
+
+  /// <summary>
+  ///   Max backoff for retries
+  /// </summary>
+  public TimeSpan RetryMaxBackoff { get; set; } = TimeSpan.FromSeconds(30);
```
Comment (aneojgurhem) [inline Client/src/Common/Properties.cs:341]: only get ?

Comment (dbrasseur-aneo) [reply Client/src/Common/Properties.cs:341]: Like any other field, it can be set from anywhere if you like.

Comment (aneojgurhem) [reply Client/src/Common/Properties.cs:341]: I don't think it is a good idea

Comment (dbrasseur-aneo) [reply Client/src/Common/Properties.cs:341]: Why would they be different from the others ?

Comment (aneojgurhem) [reply Client/src/Common/Properties.cs:341]: because we need to do the rigth things at some point

**Comment by dbrasseur-aneo** [comment]
I cherry picked a nunit fix that caused a lot of CI timeout

**Pull request merged**